be-traveltrove/src/main/java/com/traveltrove/betraveltrove/utils/DatabaseLoader.java

```diff
@@ -157,4 +159,40 @@ public void loadEvents() {
                 );
     }
 
+    // Tours
+    private final TourRepository tourRepository;
+
+    @PostConstruct
+    public void loadTours() {
+        List<Tour> sampleTours = List.of(
+                Tour.builder()
+                        .id(null)
+                        .tourId("TOUR01")
+                        .name("Spring Festival")
+                        .description("A vibrant festival celebrating the arrival of spring.")
+                        .build(),
+                Tour.builder()
+                        .id(null)
+                        .tourId("TOUR02")
+                        .name("Tech Expo 2024")
+                        .description("Showcasing the latest advancements in technology.")
+                        .build(),
+                Tour.builder()
+                        .id(null)
+                        .tourId("TOUR03")
+                        .name("Wine Tasting Retreat")
+                        .description("A relaxing getaway featuring fine wines and scenic views.")
+                        .build()
+        );
```

Comment:
Since the `toTourEntity` method in the `EntityModelUtil` class is generating UUIDs for the `tourId`, isn't it better to use UUIDs here as well for consistency?